tox.ini

```diff
@@ -12,8 +12,28 @@ deps =
     ansible22: ansible~=2.2
 
 commands =
+    unit: pytest
     flake8: flake8
     pylint: python setup.py lint
     yamllint: python setup.py yamllint
-    unit: nosetests
     generate_validation: python setup.py generate_validation
+
+[pytest]
+testpaths =
+    # TODO(rhcarvalho): enable these after https://github.com/openshift/openshift-ansible/pull/3414 merges.
+    #roles/lib_openshift/src/test/unit
+    #roles/lib_utils/src/test/unit
+    roles/openshift_master_facts/test
```

Comment:
How much time are we saving on test discovery? If it's significant then it might not be worth it. If it's only saving a second or less of execution time, then I think the benefit outweighs the cost.

We could also tie into setup tools like we do for yamllint, etc